src/interfaces/UBA.ts

```diff
@@ -19,9 +19,9 @@ export const isUbaOutflow = (flow: UbaFlow): flow is UbaOutflow => {
 };
 
 export const outflowIsFill = (outflow: UbaOutflow): outflow is FillWithBlock => {
-  return (outflow as FillWithBlock).isSlowRelay !== undefined;
+  return (outflow as FillWithBlock)?.updatableRelayData?.isSlowRelay !== undefined;
```

Comment:
I've proposed a backwards-compatible change here: https://github.com/across-protocol/sdk-v2/pull/179. It can also be merged into master to align the two.